includes/User.php

```diff
@@ -395,9 +397,6 @@ public function loadFromId() {
 			if (isset($data[self::CACHE_PREFERENCES_KEY])) {
 				$this->userPreferences()->setPreferencesInCache($this->mId, $data[self::CACHE_PREFERENCES_KEY]);
 			}
-			if (isset($data[self::CACHE_ATTRIBUTES_KEY])) {
-				$this->userAttributes()->setAttributesInCache($this->mId, $data[self::CACHE_ATTRIBUTES_KEY]);
```

Comment:
So before we were having the `$this->userAttributes()` manage attributes much more actively. Any call to `User::getGetAttribute()` would have `$this->userAttributes()` check it's own cache to see if an attribute was available and return null if not. Because of that, if we loaded a user object from cache, we needed to make sure it's corresponding `$this->userAttributes()` was loaded as well.

Going forward `$this->userAttributes()` is taking a much more background role and is only in charge of loading the attributes initially. Those are then stored in the member variable `mOptions` [here](https://github.com/Wikia/app/pull/8216/files#diff-897fba1dd6b8edc3238762e35d0b7c71R4927) which is where attributes are traditionally stored so the user object can continue to take care of all of the caching it wants. So this line isn't necessary any more.